### ISSUE TITLE
Unable to resolve symbol: inst? in this context, compiling:(prone/prep.clj:85:5)

### ISSUE THREAD

**Issue by ertugrulcetin** [magnars/prone#48]
I've got `1.5.1` prone version and getting this exception:

```
Unable to resolve symbol: inst? in this context, compiling:(prone/prep.clj:85:5)
Caused by: java.lang.RuntimeException: Unable to resolve symbol: inst? in this context
```

**Comment by magnars** [comment]
Turns out that `inst?` is new in Clojure 1.9. If you can, upgrading should solve the issue. I'll take a look at backporting it.

**Comment by ertugrulcetin** [comment]
Thank you for the update

**Comment by magnars** [comment]
Fixed and released as `1.5.2`.
